sdg/components.py

```diff
@@ -453,6 +457,8 @@ def set_precomputed_skills_data_ratio(sampling_size: float, skills_recipe: str):
                         pipeline=pipeline,
                         chunk_word_count=1000,
                         server_ctx_size=4096,
+                        batch_size=sdg_batch_size,
+                        num_cpus=sdg_num_workers,
```

Comment:
Can we follow the same naming pattern for the parameters? It makes it easier to match the parameter value passing throughout the functions